package.json

```
@@ -18,7 +18,7 @@
     "lint-js": "tsc --noEmit && yarn lint-js-eslint && yarn lint-js-prettier",
     "hooks-add": "husky install",
     "hooks-remove": "husky uninstall",
-    "start": "concurrently --kill-others --raw 'vite --host | grep -v 3000' 'yarn serve'",
+    "start": "concurrently --kill-others --raw 'vite --host | grep -v http:' 'yarn serve'",
```

Can we remove the `grep` from this command? I don't see why we would need it, and actually that was masking the issue of the "port already in use" for me today. For example, I just tried removing it from my system (where port 3000 is in use) and I got this output:
```
$ concurrently --kill-others --raw 'vite --host' 'yarn serve'
$ ./entrypoint
booting on https://localhost:8407
Port 3000 is in use, trying another one...

  VITE v6.0.7  ready in 107 ms

  ➜  Local:   http://localhost:3001/
...
```
If I had seen the filtered out line, that would have probably saved us one hour trying to figure out what was the issue today. :)
```suggestion
    "start": "concurrently --kill-others --raw 'vite --host' 'yarn serve'",
```

---

Actually, forget about this. Your edit makes the "Port already in use" line appear (which is desirable) and just filters out the misleading Vite URLs. So it's fine to keep it this way. Marking as resolved.